blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/GenericElasticSearchIO.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2013 Rackspace
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.rackspacecloud.blueflood.io;
+
+import java.util.List;
+import java.util.Map;
+
+public interface GenericElasticSearchIO {
```

Comment:
`GenericElasticSearchIO` name is a bit confusing because we already have `DiscoveryIO` which gets implemented by `ElasticIO` and does a lot of ES related operations. What do you think about changing it to `EventsIO` or something like that?